### PR TITLE
Setup Bigtable instance cleaner

### PR DESCRIPTION
**Pull request by Abacn**
It constantly get leaked instance since bigtable integration test have setup. However, the bigtable cli (cbt) does not support fetch the creation date of instances, so we have to encode this into the instance name for the tests, and delete instances of >5d age.

**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by Abacn** [comment]
Tested by run `Task :beam-test-tools:removeStaleCbtInstances` locally

**Comment by Abacn** [comment]
Run Python_Xlang_Gcp_Direct PostCommit

**Comment by Abacn** [comment]
Run Python_Xlang_Gcp_Direct PostCommit

**Comment by Abacn** [comment]
R: @ahmedabu98

File: .test-infra/tools/stale_cbt_instances_cleaner.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    pattern=$template-"([0-9]{8})"-
+    if [[ $instance =~ $pattern ]]; then
+      CREATE_DATE=${BASH_REMATCH[1]}
+      # skip if not a valid date
+      CREATED=`date -ju -f "%Y%m%d-%H%M%S" ${CREATE_DATE}-000000 +%s` || continue
+      if [[ $GRACE_PERIOD -gt $CREATED ]]; then
+        if cbt -project=$PROJECT deleteinstance $instance; then
+          echo "Deleted $instance (created $CREATE_DATE)"
+        else
+          failed_calls+=1
```
Comment (ahmedabu98) [inline .test-infra/tools/stale_cbt_instances_cleaner.sh:46]: Maybe we should also log failed deletions for better debugging

**Review by ahmedabu98 — APPROVED** [review]
Just one comment but LGTM

File: .test-infra/tools/stale_cbt_instances_cleaner.sh (hunk 1 above)
Comment (Abacn) [reply .test-infra/tools/stale_cbt_instances_cleaner.sh:46]: Thanks for the response. When it fails the deletion command (`cbt -project=$PROJECT deleteinstance $instance`) itself will print error message for each instance. So this is just a summary at the end

**Pull request merged**